gh-pr-context.py

```
@@ -316,17 +316,35 @@ def cmd_logs(argv):
+        check_run_id = run["id"]
         name = run["name"]
-        log_content = ""
+
+        # Resolve real job IDs from the check-run (check-run IDs ≠ job IDs)
+        job_ids = []
         try:
-            args = _gh_cmd() + ["api", f"repos/{owner_repo}/actions/jobs/{job_id}/logs"]
-            result = subprocess.run(args, capture_output=True, text=True)
-            if result.returncode == 0:
-                log_content = result.stdout
+            jobs_args = _gh_cmd() + ["api", f"repos/{owner_repo}/check-runs/{check_run_id}/jobs"]
```

Ruff (0.15.15)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)